### PR TITLE
Log a warning if a fuel station detail couldn't be fetched

### PR DESCRIPTION
**Pull request by TheDuffman85**
Log a warning with helpful information if a fuel station detail could not be fetched. In my case I used a station id for a fuel station that is no longer valid. Throwing a error without further information was confusing. I think logging a warning is the better approach here.

grafik

### CODE REVIEW

File: apis/tankerkoenig.js [hunk 1]
```diff
@@ -186,7 +186,8 @@ async function setStationInfos(stationsByRadius) {
         const parsedResponse = await response.json();
 
         if (!parsedResponse.ok) {
-            throw new Error('Error no fuel station detail');
+            console.warn(`No fuel station detail. StationId: ` + stationId + ` Error: ` + parsedResponse.message);
```
Comment (fewieden) [inline apis/tankerkoenig.js:189]:
```suggestion
            console.warn(`No fuel station detail. StationId: ${stationId} Error: ${parsedResponse.message}`);
```

**Comment by fewieden** [comment]
@TheDuffman85 there seems to be another issue with the station id list. It is fixed by this change, but I think we should change the behavior anyways as this goes unnecessary against the API limit. If the station id is in the radius search and id search and we filter out the whole list, then the request should not be made at all. I suggest to also add a warning for this that the list is empty after filtering duplicated stations from the radius search, see issue https://forum.magicmirror.builders/post/90310.

Do you want to handle it or should I fix it?

**Comment by TheDuffman85** [comment]
@fewieden I can look into this tomorrow. But I also wouldn't mind if you fix it.

**Comment by fewieden** [comment]
@TheDuffman85 I will take over then in the next few days, I also found outdated documentation that I need to fix as well.

**Pull request merged**